doc/release/technotes/README.libraries.rst

```diff
@@ -23,36 +26,111 @@ the ``chpl`` command line take precedence over any found through object
 library paths (``-L``).  Where there is a conflict, the last library
 specified takes precedence.
 
-When Chapel code is accessed as library code, it is the user's responsibility to
-ensure that modules are initialized before they are used.  For a module
-named ``<myModule>``, the corresponding initialization routine is
-named ``chpl__init_<myModule>``.  Therefore, for example, a C program that
-makes use of the standard Chapel module ``Random`` should
-call ``chpl__init_Random`` at some point before the Random interface is first
-used.
+When creating a library file from Chapel code, only those symbols with
+``export`` attached to them will be available from outside the library.  For
+example:
+
+.. code-block:: Chapel
+
+   // This function will be available to the library user
+   export proc foo() {
+     // Does something
+     ...
+   }
+
+   // but this function will not be
+   proc bar() {
+     // Does something else
+     ...
+   }
```

Comment:
Could you name this bar() method something else, so it isn't confusingly named the same as the bar/baz finding .c examples below?